### DISCUSSION TITLE
Table export function corrupting special character

### DISCUSSION THREAD

**Discussion by Dods-A** [primefaces/community#823] [PrimeNG] [answered]
I'm currently using v13 and was able to replicate this issue in the stack blitz example provide in the v13 and v14 docs. I was unable to replicate in v17 as there seems to be an issue with the stackblitz example.

When exporting the em dash — (not -) it is replaced with "â€”"
![image](https://github.com/primefaces/community/assets/150770527/6a351228-3c86-449b-b6fd-6b13af9f3c16)

**Comment by cetincakiroglu** [comment]
Hi,

v13 is an outdated version and we cannot provide a fix for the legacy versions. Unfortunately, we need to replicate the issue to understand and fix it. So right now we can't take action on the case. Please provide an example if it's possible.

**Reply by Dods-A** [reply]
Are you able to see that the stackblitz link for table exporting in the current documentation is not running? there are issues with the file-saver import

After fixing the demo code i was still able to replicate this issue using v17
https://stackblitz.com/edit/mryvge?file=src%2Fservice%2Fproductservice.ts

**Reply by Dods-A** [reply]
@cetincakiroglu any luck replicating using the stackblitz link?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

I couldn't reproduce the behavior.

![Screenshot 2024-01-13 at 13 46 57](https://github.com/primefaces/community/assets/69278826/b3b58c9d-e295-409e-9a02-91a51c758efe)

**Reply by Dods-A** [reply]
Sorry it appears like this is actually an Excel issue. Thanks for helping to figure this out.
